app/src/main/res/values/styles.xml

```diff
@@ -58,12 +58,6 @@
         <item name="android:textAllCaps">false</item>
     </style>
 
-
-    <style name="Widget.Ping.TextInputLayout" parent="Widget.MaterialComponents.TextInputLayout.OutlinedBox">
-        <item name="boxStrokeColor">?android:textColorSecondary</item>
-        <item name="hintTextColor">?android:textColorSecondary</item>
-    </style>
```

Comment:
Was this reported as unused? It's just to know if this requires testing (esp. on older devices).

Reply:
it was reported as unused :)